enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionDefaultView/DefaultViewTitle.tsx

```diff
@@ -27,7 +27,7 @@ const DefaultViewTitleText = ({
   <Stack gap="xs">
     {originalName && withResetButton && isQuestionChanged && (
       <Text fw={600} size="sm">
-        Return to
+        {t`Return to`}
```

Comment:
This should be a whole sentence, so the anchor should appear inside the translation, with the help of jt

A contextual string would be useful here, to explain what the variable is

I can explain more in a slack huddle if you like